rulesets/nixpkgs/require-merge-queue-except-periodic-merges.json

```
@@ -12,6 +12,7 @@
         "refs/heads/haskell-updates",
         "refs/heads/staging",
         "refs/heads/staging-next",
+        "refs/heads/staging-nixos",
         "refs/heads/staging-25.05",
         "refs/heads/staging-next-25.05"
```

After exporting it again, this looks like this:


```suggestion
        "refs/heads/staging-25.05",
        "refs/heads/staging-next-25.05",
        "refs/heads/staging-nixos"
```